roles/deploy_flask_app/files/create.yaml

```diff
@@ -0,0 +1,234 @@
+---
+- name: Set 'region' variable
+  ansible.builtin.set_fact:
+    region: "{{ region | default(aws_region) }}"
+
+- name: Create resources playbook
+
+  block:
+    - name: Get image ID to create an instance
+      amazon.aws.ec2_ami_info:
+        filters:
+          architecture: x86_64
+          virtualization-type: hvm
+          root-device-type: ebs
+          name: "{{ image_filter }}"
+      register: images
```

Comment:
I'm not sure about this one, but shouldn't ansible-lint  normally complain about how the registered variable name is called because it should start with the role_name?